dlc-wallet/src/main/scala/org/bitcoins/dlc/wallet/DLCWallet.scala

```diff
@@ -88,6 +88,7 @@ abstract class DLCWallet extends Wallet with AnyDLCHDWalletApi {
             new IllegalArgumentException(
               s"No DLCDb found with paramHash ${paramHash.hex}"))
       }
+      _ = logger.debug(s"Updating DLC to state $state")
```

Comment:
might be nice here and below to log the `state.toJson` instead of the case class serialization

Reply:
state doesn't have json, state is just like Offered, Accepted, etc.